crates/runtime/src/lib.rs

```diff
@@ -207,6 +205,15 @@ impl<Handler, Runtime: SyscallPtrAccess + HintProcessor> SyscallPtrAccess
         self.extended_runtime.verify_syscall_ptr(ptr)
     }
 }
+impl<Runtime: SyscallPtrAccess> SyscallPtrAccess for ExtendedRuntime<Runtime> {
```

Comment:
Isn't that doable with a macro btw?

Reply:
It is generic, so it will work for all structs

Reply:
There will be no more implementations of it, next PR simplifies it grately